src/dbus.c

```diff
@@ -506,7 +508,35 @@ method_try(sd_bus_message *m, void *userdata, sd_bus_error *ret_error)
                                  "cannot watch 'netplan try' child: %s", strerror(-r));
         // LCOV_EXCL_STOP
 
-    return sd_bus_reply_method_return(m, "b", true);
+    /* wait for output from the 'netplan try' subprocess, that indicates it is
+     * ready to accept any Accept/Reject signal from us (e.g. SIGTERM/SIGUSR1). */
+    char buf[512];
+    const char* buf_p = (char*)&buf;
+    struct pollfd fd = { .fd = child_stdout, .events = POLLIN };
+    guint poll_timeout = 5000;
+    if (seconds > 0 && seconds < 5)
+        poll_timeout = seconds * 1000;
+    /* Timeout after up to 5 sec of waiting on stdout */
+    while (poll(&fd, 1, poll_timeout) > 0) {
+        ssize_t cnt = read(child_stdout, &buf, 512);
```

Comment:
(nitpick²) this will not detect short reads and if that happens thing will fail with the timeout  - which I think are probably not a problem here but wanted to mention it anyway.